### ISSUE TITLE
hydration reminder push notifications

### ISSUE THREAD

**Issue by anandijain** [free2pee/Free2Pee.jl#11]
@acoh64 can we get a hydration reminder? this was recommended by @MarcBerliner. we don't want to just dehydrate our users, we want them to be completely full of piss! 

@free2pee/kernel-team @ilanhaskel

**Comment by acoh64** [comment]
I agree. Also, the more hydrated people are, the more they will pee, and thus the more crowd sourcing we will have to build the data set.
